frontend/src/main/pages/SectionSearch/SectionSearchIndexPage.js

```diff
@@ -1,14 +1,41 @@
+import { useState } from "react";
 import BasicLayout from "main/layouts/BasicLayout/BasicLayout";
+import BasicCourseSearchForm from "main/components/BasicCourseSearch/BasicCourseSearchForm";
+import { useBackendMutation } from "main/utils/useBackend";
 
 export default function SectionSearchIndexPage() {
+  // Stryker disable all
+  const [_sectionJSON, setSectionJSON] = useState([]);
+  const objectToAxiosParams = (query) => ({
+    url: "/api/sections/basicsearch",
+    params: {
+      quarter: query.quarter,
+      subjectArea: query.subject,
+      courseLevel: query.level,
+    },
+  });
+
+  const onSuccess = (section) => {
+    console.log()
+    setSectionJSON(section);
+  };
+
+  const mutation = useBackendMutation(
+    objectToAxiosParams,
+    { onSuccess },
+    []
+  );
+    //Stryker enable all
+  async function fetchBasicCourseJSON(_event, query) {
+    mutation.mutate(query);
+  }
   return (
     <BasicLayout>
       <div className="pt-2">
-        <h1>Section Search</h1>
-        <p>
-          This is where the index page will go
-        </p>
+        <h5>Section Search</h5>
+        <BasicCourseSearchForm fetchJSON={fetchBasicCourseJSON} />
+        {/* <SectionsTable sections={sectionJSON} /> */}
```

Comment:
Why is this commented out?

Reply:
Typically, one would get that SectionTable done and merged first, and then merge this.

Otherwise, there's very little point; it's not possible to test all of the other code on the page without that in place.